test_pii_data.py

```diff
@@ -50,7 +50,7 @@ def test_has_us_phone(self):
         self.assertFalse(test_data.has_us_phone())
 
     def test_has_email(self):
-        test_data = Pii()
+        test_data = Pii('My email is kavondean@gmail.com')
         self.assertEqual(test_data.has_email(), None)
```

Comment:
Does the unit test run OK?  This doesn't look right, I would expect your has_email() to return True not None